tests/installation/setup_zdup.pm

```diff
@@ -19,8 +19,8 @@ use utils;
 sub run() {
     my ($self) = @_;
 
-    $self->wait_boot(ready_time => 600);
-
+    assert_screen 'displaymanager', 500;
+    ensure_unlocked_desktop;
```

Comment:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2849 and https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2850 are my two proposals